tests/test_awssh.py

```diff
@@ -30,14 +23,14 @@
 
 # define sources of version strings
 RELEASE_TAG = os.getenv("RELEASE_TAG")
-PROJECT_VERSION = example.__version__
+PROJECT_VERSION = awssh.__version__
 
 
 def test_stdout_version(capsys):
     """Verify that version string sent to stdout agrees with the module version."""
     with pytest.raises(SystemExit):
         with patch.object(sys, "argv", ["bogus", "--version"]):
-            example.example.main()
+            awssh.main()
```

Comment:
In the skeleton the `example.example` package is brought into the namespace [here](https://github.com/cisagov/skeleton-python-library/blob/20920997de8b9c82e89013ea96abab690a56633d/src/example/__init__.py#L7). Nothing similar happens in this project (and cannot because you will end up with a circular import).

Reply:
@mcdonnnj - Thanks for pointing that out- makes sense. 👍 
@felddy - If you're satisfied here, please resolve this conversation.